openedxstats/settings/testing.py

```diff
@@ -1,3 +1,15 @@
 from .base import *  # noqa
 
 # By default, nothing is different for testing.
+
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```

Comment:
The README implied that postgres is optional for tests; if that's inaccurate, we should either fix the tests or update the README.